components/script/dom/create.rs

```diff
@@ -308,6 +308,8 @@ pub fn create_native_html_element(
         // https://html.spec.whatwg.org/multipage/#other-elements,-attributes-and-apis:isindex-2
         local_name!("isindex") => make!(HTMLUnknownElement),
         local_name!("kbd") => make!(HTMLElement),
+        // https://html.spec.whatwg.org/multipage/#keygen
```

Comment:
While I tried to make the URL look like the others, it appears that the scheme had changed too much in order to allow that; even some of the old URLs aren't redirecting at all